From 6f6a7acd3de546128d09547d19d8c67c878b08b5 Mon Sep 17 00:00:00 2001
From: chrmhoffmann <chrmhoffmann@gmail.com>
Date: Wed, 1 Nov 2017 20:48:41 +0100
Subject: [PATCH 018/249] FP2: generate better wiki pages

* lower case short vendor to make links work into device tree
* rename model to FP2, so that wiki pages don't say Fairphone Fairphone 2
* remove bootloader entry, no specific version required

Change-Id: I77ed895d88b363fcafec98df67ae71d0d74b2c45
---
 _data/devices/FP2.yml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/_data/devices/FP2.yml b/_data/devices/FP2.yml
index ea9170f..1dc5214 100644
--- a/_data/devices/FP2.yml
+++ b/_data/devices/FP2.yml
@@ -22,7 +22,7 @@ install_method: fastboot_unlocked
 kernel: android_kernel_fairphone_msm8974
 maintainers: [chrmhoffmann]
 models: [FP2]
-name: Fairphone 2
+name: FP2
 network:
 - {tech: 2G, bands: '850 900 1800 1900' }
 - {tech: 3G, bands: '8(900) 2(1900) 1(2100) ' }
@@ -31,7 +31,6 @@ peripherals: [3-axis Compass, 3D Accelerometer, 3D Gyroscope, Ambient light, Dua
 ram: 2 GB
 recovery_boot: With the device powered off, hold Volume Up & Power. Release when boot logo appears.
 release: December 2015
-required_bootloader: []
 screen: 130 mm (5.1 in)
 screen_ppi: 446
 screen_res: 1920x1080
@@ -42,7 +41,7 @@ storage: 32 GB
 tree: android_device_fairphone_FP2
 type: phone
 vendor: Fairphone
-vendor_short: Fairphone
+vendor_short: fairphone
 versions: [14.1]
 width: 73 mm (2.9 in)
 wifi: 802.11 b/g/n/ac
-- 
2.17.1

